common/containers/Tabs/GenerateWallet/components/PaperWallet.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import translate from 'translations';
+import PrintableWallet from 'components/PrintableWallet';
+
+export default class PaperWallet extends Component {
+  static propTypes = {
+    // State
```

Comment:
I get what this means now. I think it'd be better written as `// Store`, not state.